### PR TITLE
fix addons broken due to GRASS 8 modules API

### LINKED ISSUE #616: [Bug] Some AddOns broken for GRASS 8 due to their API change
After the branch for GRASS 8 will be finally created (see #528), fix addons affected by this change. 

The PR will tag original authors of the addons as reviewers to give them a chance to state their opinion about changes.

So far those addons were discovered to need some care:

- [x] i.sentinel.coverage - caused by a change in v.db.select
- [x] v.habitat.dem - caused by a change in v.db.select
- [x] r.valley.bottom (temporarily fixed by an if statement) - going to have a PR for `grass7` and `grass8` branch - caused by a change in r.neighbors

### PR DESCRIPTION
**Pull request by pesekon2**
fix #616

### CODE REVIEW

File: src/vector/v.habitat.dem/v.habitat.dem.py [hunk 1]
```diff
@@ -285,7 +277,7 @@ def main():
     grass.run_command(
         "v.db.select",
         map=v_habitat,
-        flags="v",
+        format="vertical",
```
Comment (hellik) [inline src/vector/v.habitat.dem/v.habitat.dem.py:280]: Looks good

File: src/vector/v.habitat.dem/v.habitat.dem.py [hunk 2]
```diff
@@ -296,7 +288,7 @@ def main():
...
```
Comment (hellik) [inline src/vector/v.habitat.dem/v.habitat.dem.py:291]: Looks good

**Comment by hellik** [comment]
Changes checked for v.habitat.dem, looks fine. Thanks for updaating.

File: src/vector/v.habitat.dem/v.habitat.dem.py (hunk 2 above)
Comment (pesekon2) [reply src/vector/v.habitat.dem/v.habitat.dem.py:291]: Thanks for the review.

**Pull request merged**